cinema/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
```

Comment:
The import statement should be followed by a newline for better readability. Consider adding a newline after this line.